### PR TITLE
FEAT-#1847: High performance, no shuffle train_test_split

### LINKED ISSUE #1847: No shuffle train_test_split experimental API
**Is your feature request related to a problem? Please describe.**
A clear and concise description of what the problem is. What kind of performance improvements would you like to see with this new API?

A higher performing `train_test_split` will improve end to end performance time with machine learning pipelines.

### PR DESCRIPTION
**Pull request by devin-petersohn**
Signed-off-by: Devin Petersohn <devin.petersohn@gmail.com>

## What do these changes do?

- [x] commit message follows format outlined [here](https://modin.readthedocs.io/en/latest/contributing.html)
- [x] passes `flake8 modin`
- [x] passes `black --check modin`
- [x] signed commit with `git commit -s` 
- [x] Resolves #1847 
- [ ] tests added and passing

### CODE REVIEW

File: modin/experimental/sklearn/model_selection/train_test_split.py [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+# the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF
+# ANY KIND, either express or implied. See the License for the specific language
+# governing permissions and limitations under the License.
+
+
+def train_test_split(df, **options):
+    train_size = options.get("train_size", 0.75)
+    train = df.iloc[: int(len(df) * train_size)]
+    test = df.iloc[len(train) :]
+    return train, test
```
Comment (anmyachev) [inline modin/experimental/sklearn/model_selection/train_test_split.py:19]:
```suggestion
import warnings


def train_test_split(*arrays, **options):
    warnings.warn("Modin' implementation of train_test_split support only 'train_size' option")
    result = []
    train_size = options.get("train_size", 0.75)
    for df in arrays:
        result.append(df.iloc[: int(len(df) * train_size)])
        result.append(df.iloc[len(train) :])
    return result
```

Comment (anmyachev) [reply modin/experimental/sklearn/model_selection/train_test_split.py:19]: Please note, the changes is not tested by me.

**Comment by anmyachev** [comment]
@devin-petersohn LGTM, assuming CI is passed.

**Review by anmyachev — APPROVED** [review]
